### ISSUE TITLE
Markdown link to relative image?

### ISSUE THREAD

**Issue by reagle** [alvesjtiago/hover-preview#38]
The following does not seem to work. Should it?

```
![caption](media/pixabay-dog-115078_640.jpg)
```

**Comment by AmjadHD** [comment]
- make sure there is a 'media' folder with that image in it, inside your project folder.
- if you switched 'recursive' to false and/or 'search_mode' to 'file' in the settings, try switching them back to their defaults.

**Comment by reagle** [comment]
I don't use projects, which makes me think I should use the `file` `search_mode.` However, there is no `Hover Preview.sublime-settings` file in my install. Nor is there an entry in `/Sublime/Preferences/Package Settings` ...?

**Comment by AmjadHD** [comment]
For that you need to create a `Hover Preview.sublime-settings` in `Packages/User` folder.
- you can use this [patched version](https://github.com/AmjadHD/sublime-edit-settings.git) of the default `settings.py`, this way you can easily edit your preferences:

### Manually
1. create a folder named `Default` in you Packages directory
2. place the patched `settings.py` in it
3. add
```json
    {
        "caption": "Edit",
        "command": "edit_settings"
    },
```
to your `Default.sublime-commands`

### With Package Resource Viewer
this process becomes easier with the [Package Resource Viewer](https://packagecontrol.io/packages/PackageResourceViewer) package
1. open command palette
2. select `PackageResourceViewer: Open Resource`
3. select `Default` > `settings.py`
4. replace by the content of the patched file and save (a folder named `Default` that contains the patched `settings.py` is then automatically created in your Packages directory)
5. repeat the process to edit `Default.sublime-commands`

## Usage

1. open the command palette
2. select Edit > settings > Hover Preview

**Comment by reagle** [comment]
Okay, I created the file manually and it works. Why not include the setting file as part of the package install?

**Comment by AmjadHD** [comment]
#37 adds an entry in the package settings.

**Comment by alvesjtiago** [comment]
This is addressed with #37 and version 0.1.0 which should be automatically updated via package manager.
